hosted/gke/k8s_chart_support/k8s_chart_support_provisioning_test.go

```diff
@@ -0,0 +1,54 @@
+package k8s_chart_support_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/clusters/gke"
+	"github.com/rancher/shepherd/pkg/config"
+
+	"github.com/rancher/hosted-providers-e2e/hosted/gke/helper"
+	"github.com/rancher/hosted-providers-e2e/hosted/helpers"
+)
+
+var _ = Describe("K8sChartSupportProvisioning", func() {
+
+	var (
+		cluster *management.Cluster
+	)
+	BeforeEach(func() {
+		gkeConfig := new(management.GKEClusterConfigSpec)
+		config.LoadAndUpdateConfig(gke.GKEClusterConfigConfigurationFileKey, gkeConfig, func() {
+			gkeConfig.ProjectID = project
+			gkeConfig.Zone = zone
+			labels := helper.GetLabels()
+			gkeConfig.Labels = &labels
+			gkeConfig.KubernetesVersion = &k8sVersion
+			for _, np := range gkeConfig.NodePools {
+				np.Version = &k8sVersion
+			}
+		})
+
+		var err error
+		cluster, err = gke.CreateGKEHostedCluster(ctx.RancherClient, clusterName, ctx.CloudCred.ID, false, false, false, false, map[string]string{})
+		Expect(err).To(BeNil())
+		cluster, err = helpers.WaitUntilClusterIsReady(cluster, ctx.RancherClient)
+		Expect(err).To(BeNil())
+	})
+
+	AfterEach(func() {
+		if ctx.ClusterCleanup {
+			err := helper.DeleteGKEHostCluster(cluster, ctx.RancherClient)
+			Expect(err).To(BeNil())
+		} else {
+			fmt.Println("Skipping downstream cluster deletion: ", clusterName)
+		}
+	})
+
+	It(fmt.Sprintf("should successfully test k8s %s chart support on rancher %s", k8sUpgradedMinorVersion, rancherUpgradedVersion), func() {
+		commonChartSupportUpgrade(&ctx, cluster, clusterName, rancherUpgradedVersion, helpers.RancherHostname, k8sUpgradedMinorVersion)
+	})
+
```

Comment:
It(fmt.Sprintf("should successfully test k8s %s chart support **provisioning** on **upgraded** rancher %s", k8sUpgradedMinorVersion, rancherUpgradedVersion)

This TC will also be required to automated, can you add  TODO ?

Reply:
Done.